src/Microsoft.Health.Dicom.Core/Features/Validation/ValidationErrorCode.cs

```diff
@@ -0,0 +1,65 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.Validation
+{
+    /// <summary>
+    /// Validation Error Code.
+    /// Error Code is  4 letter number. 
+    /// First 2 letters indicate VR:
+    ///  00 - General error for all VR
+    ///  01 - Error for PN
+    /// Last 2 letters indicate specific errors for VR.
```

Comment:
I think it makes sense to try to group error codes together by VR, but I think if we want to do this we need to rephrase it. Ultimately, the values we operate upon will be integral numbers and not strings. Therefore, there won't be a concept of leading 0s. I think the more correct structure, if we so choose to do so, would be to organize them into ranges. Eg [0, 100)  are general errors, [100, 101) are errors for VR type 1, etc. (Or however large we want the ranges to be)